modules/product/product.service.js

```diff
@@ -85,6 +86,14 @@ class ProductsService {
     };
   }
 
+  async updateProduct(id, products) {
+    const product = await Products.findById(id);
```

Comment:
I think we need this check here too
`if (await this.checkProductExist(data)) {
      throw new Error(PRODUCT_ALREADY_EXIST);
}`

Reply:
done